### PR TITLE
code refactoring

### PR DESCRIPTION
**Pull request by tomhsiao1260**
I did some code refactoring to make it easier to understand (the output interface has not changed). Feel free to let me know anything inappropriate and I'll change them back. Thanks!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,72 +1,98 @@
-import { Scene, PerspectiveCamera, WebGLRenderer, Color, TorusKnotGeometry, SphereGeometry, FontLoader, TextBufferGeometry } from 'three';
-import { OrbitControls } from "three/examples/jsm/controls/OrbitControls";
+import * as THREE from 'three';
```
Comment (PWhiddy) [inline index.js]: I think we prefer explicit imports over importing all under a namespace, ie `import { thing } from "three"` preferred to ` import * as THREE from "three" `

File: index.js [hunk 2]
```diff
@@ -1,72 +1,98 @@
...
-    _scale: scale
+    _scale: params.scale
 } ));
 
-
+state.mesh = mesh;
 scene.add(mesh);
 
-if( 'text' in qParams) {
+// const pane = new Pane();
```
Comment (PWhiddy) [inline index.js:84]: @torinmb if these are commented out can we just remove them?

File: src/editor.js [hunk 3]
```diff
@@ -1,40 +1,40 @@
-import {EditorState, basicSetup} from "@codemirror/basic-setup"
-import {EditorView, keymap} from "@codemirror/view"
-import {indentWithTab} from "@codemirror/commands"
-import {javascript} from "@codemirror/lang-javascript"
+import { EditorState, basicSetup } from '@codemirror/basic-setup'
```
Comment (PWhiddy) [inline src/editor.js]: I think we're moving to prefer double quotes over single quotes for strings, and formatted strings using backticks when values need to be injected.

**Review by PWhiddy — CHANGES_REQUESTED** [review]
Hi, thanks for this! 
Most of the organization improvements look good to me. 
I left a couple small notes on style, some of which we are not currently following everywhere yet but I think that's the direction we'd like to move in.
With those changes I can merge!

File: index.js (hunk 1 above)
Comment (tomhsiao1260) [reply index.js]: Hi, thanks for letting me know this, I've changed it back: 2e9ce944051a93b90541acdd85431cc5d3cb12c2

File: src/editor.js (hunk 3 above)
Comment (tomhsiao1260) [reply src/editor.js]: Hi, there is the change: 8fc4d42356672c6825a8b37e990d36a86505f77c.

**Comment by PWhiddy** [comment]
Awesome! 
I'm going to go ahead and merge this, but if you see any issues @torinmb lmk. We need to finish the multipass feature branch, more commits on main mean more rebasing.

**Pull request merged**